requirements.txt

```diff
@@ -2,32 +2,32 @@
 
 # If you update these, also update iati_datastore/setup.py
 # These are duplicated here so that requires.io will flag out of date dependencies
-Flask==0.10
-Jinja2==2.6
-SQLAlchemy==0.8
-Flask-SQLAlchemy==1.0
-Werkzeug==0.8.3
-ckanapi==1.5
-iso8601==0.1.4
-lxml==3.4.1
-psycopg2==2.4.6
-python-dateutil==2.1
-six==1.10.0
-unicodecsv==0.9.4
-voluptuous==0.7.1
-Flask-Script==0.5.3
-prettytable==0.7
-alembic==0.5.0
-gunicorn==0.17.2
-defusedxml==0.4
-redis==2.7.2
-rq==0.3.7
-Unidecode==0.04.12
-requests==2.5.3
-Flask-RQ==0.2
-flask-heroku==0.1.4
-Flask-And-Redis==0.4
-Flask-Markdown==0.3
-xmltodict==0.7.0
-gevent>=0.13.8
+Flask==0.10  # rq.filter: <0.11
+Jinja2==2.6  # rq.filter: <3.0
+SQLAlchemy==0.8  # rq.filter: <0.9
+Flask-SQLAlchemy==1.0  # rq.filter: <2.0
+Werkzeug==0.8.3  # rq.filter: <0.9
+ckanapi==1.5  # rq.filter: <2.0
+iso8601==0.1.4  # rq.filter: <0.2
+lxml==3.4.1  # rq.filter: <4.0
+psycopg2==2.4.6  # rq.filter: <3.0
+python-dateutil==2.1  # rq.filter: <3.0
+six==1.10.0  # rq.filter: <2.0
+unicodecsv==0.9.4  # rq.filter: <0.10
+voluptuous==0.7.1  # rq.filter: <0.8
```

Comment:
Yeah, so… This will cause requirements.io to try to upgrade to 0.7.2, which appears to be completely broken. So you can either pin 0.7.1 here, or try a much newer version.

Reply:
Suggest pinning to `0.7.1`, or manually installing `0.7.2` (or even `0.10.5`). Then run the tests to see how if it works...

N.B. voluptuous is used in `validators.activity_api_args` ([link](https://github.com/IATI/IATI-Datastore/blob/975d625304e3fe615ad15ddcb8cc069cecdb21f6/iati_datastore/iatilib/frontend/validators.py)), which is tested [here](https://github.com/IATI/IATI-Datastore/blob/975d625304e3fe615ad15ddcb8cc069cecdb21f6/iati_datastore/iatilib/test/test_validators.py).